packages/traceloop-sdk/traceloop/sdk/dataset/attachment.py

```
@@ -0,0 +1,285 @@
+        elif self.content_type.startswith("audio/"):
+            return FileCellType.AUDIO
+        else:
+            return FileCellType.FILE
+
+    def _get_file_data(self) -> bytes:
+        """Get file data as bytes."""
+        if self.data:
+            return self.data
+        elif self.file_path:
+            if not os.path.exists(self.file_path):
+                raise FileNotFoundError(f"File not found: {self.file_path}")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)